german/es-wird-kalt/Kälte

```diff
@@ -0,0 +1,3 @@
+Die Beine werden langsam kalt. Sie wissen, dass Sie sich bewegen müssen.
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.